Library/Homebrew/version.rb

```diff
@@ -160,7 +180,7 @@ def <=>(other)
       when StringToken
         1
       when NullToken
-        -Integer(other <=> self)
+        -T.must(other <=> self)
```

Comment:
Shouldn't this be `nil`, since it's not comparable?

Reply:
See reply below (in Conversation tab) / above (in Files changed tab)